lib/wsdl.js

```diff
@@ -868,11 +867,21 @@ ElementElement.prototype.description = function(definitions, xmlns) {
 
         var elem = {};
         definitions.descriptions.types[typeName] = elem;
+        var description = typeElement.description(definitions, xmlns);
+        if (typeof description === 'string') {
+          elem = description;
+        }
+        else {
+          for (var key in description) {
+            elem[key] = description[key];
```

Comment:
Shouldn't we check if `key` is actually not inherited?

I think of either checking for the property wihtin the `for` loop with `description.hasOwnProperty(key)` or just iterating over the object keys wiht 
```javscript
Object.keys(description).forEach(function (key) {
  elem[key] = description[key];
});
```